File: PRAKTIKUM_2/soal5.py
from math import sqrt
a=float(input("masukan nilai sisi tegak segitiga : "))
b=float(input("masukan nilai sisi miring segitiga : "))
alas=sqrt((b*b)-(a*a))
keliling=a+b+alas
luas=0.5*alas*a
print(f"Alas = {alas:.0f} cm\nTinggi = {a:.0f} cm\nKeliling = {keliling:.0f} cm\nLuas = {luas:.0f} cm^2")

File: PRAKTIKUM_2/soal5.c
#include <stdio.h>
#include <math.h>
int main(){
    int a,b,alas,keliling,luas;
    printf("masukan sisi tegak segitiga : ");
    scanf("%d",&a);
    printf("masukan sisi miring segitiga: ");
    scanf("%d",&b);
    alas=sqrt((b*b)-(a*a));
    keliling=a+b+alas;
    luas=0.5*alas*a;
    printf("Alas = %d cm\nTinggi = %d cm\nKeliling = %d cm\nLuas = %d cm",alas,a,keliling,luas);
}